### PR TITLE
feature: unrevised revs on user history page

### PR DESCRIPTION
**Pull request by elbotho**
### First draft
The "activity" page for users now has a section with relevant unrevised revisions.
This is accessible through the user menu and also the profile.
Currently this visible for all users, so reviewers can also find all the unrevised revisions by that user.

### Preview
- [Example with unrevised revs](https://frontend-git-my-unrevised-serlo.vercel.app/event/history/user/141826/renate_mit_vielen_interessen) (as of today 😅 )
- [Example without](https://frontend-git-my-unrevised-serlo.vercel.app/event/history/user/26217/kulla) 

image

--- 
I will merge this into staging only after the first round of feedback by @yolo-bird and @kathongi is through.

### CODE REVIEW

**Comment by Entkenntnis** [comment]
love it :heart: it has always been difficult to manage your own edits

**Comment by elbotho** [comment]
okay peeps (@yolo-bird, @kathongi) 
it's now lives on it's own little page with a title and everything.
(menu entry can't be tested in this branch, translations will come in staging)

### Preview link:
- revisions: https://frontend-git-my-unrevised-serlo.vercel.app/entity/unrevised/user/141826/Renate_mit_vielen_Interessen
- no revisison: https://frontend-git-my-unrevised-serlo.vercel.app/entity/unrevised/user/600/maspengler

### menu entry:
image

### default view
image

### Own unrevised revisions (hides username in table)
image

### No revisions found
image

**Comment by kathongi** [comment]
Looks great! There's only one (or two) thing(s) I would change. First: unrevised revisions? Isn't it unrevised changes in the english language version?
If we call the page "unrevised changes"/"ungeprüfte Bearbeitungen" it's confusing because we already have this page with **all** unrevised changes. Can we call it "my edits "/"Meine Bearbeitungen" instead?

**Comment by elbotho** [comment]
Thanks, will check the english naming.
It's actually called `Meine Bearbeitungen` when you are on your own page (see third screenshot).

Other people can theoretically also see the edits of this user, but we don't link to this page atm. and I would say that's another step.

**Comment by elbotho** [comment]
@kathongi maybe `unrevised edits` would make sense, but we currently use `unrevised revisions` everywhere. So that's something we would need to discuss with the unit.

**Comment by kathongi** [comment]
I only checked en.serlo.org and in the menu on the top right it says "unrevised changes".
"unrevised edits" sounds good to me.

File: src/fetcher/unrevisedRevisions/query.ts [hunk 1]
```diff
@@ -1,191 +1,180 @@
 // These types are auto-generated from the GraphQL schema
 import { gql } from 'graphql-request'
 
-export const unrevisedRevisionsQuery = gql`
```
Comment (Entkenntnis) [inline src/fetcher/unrevisedRevisions/query.ts]: kebap-case: rename folder -> `/unrevised-revisions/`

File: src/data/de/index.ts [hunk 2]
```diff
@@ -355,6 +355,9 @@ export const loggedInData = {
     }, {
       url: '/event/history/user/me',
       title: "Meine Aktivitäten"
+    }, {
+      url: '/entity/unrevised/user/me',
+      title: "Ungeprüfte Bearbeitungen"
```
Comment (Entkenntnis) [inline src/data/de/index.ts:360]: this menu entry is not working - as far as I am following the unrevised revisions are back in "Meine Aktivitäten" -> removing menu entry for all languages again?

**Review by Entkenntnis — CHANGES_REQUESTED** [review]
thank you! looks great and is working fine in localhost, ready for staging. same small feedbacks.

File: src/data/de/index.ts (hunk 2 above)
Comment (elbotho) [reply src/data/de/index.ts:360]: it's part of `index.ts` so it will get overwritten by crowdin. will change other languages once it's merged.

**Comment by Entkenntnis** [comment]
Have fun on staging :)

**Comment by ghost** [comment]
Looks good. From our side it is ready to go live in production. :-)

**Comment by elbotho** [comment]
thanks @yolo-bird !

**Pull request merged**